cmd/webhook/main.go

```diff
@@ -102,7 +101,7 @@ func NewDefaultingAdmissionController(ctx context.Context, cmw configmap.Watcher
 	// above and fetched off of context by the api code.  See knative/serving's logic
 	// around config-defaults for an example of this.
 	chDefaulter := defaultchannel.New(logger.Desugar())
-	eventingduckv1alpha1.ChannelDefaulterSingleton = chDefaulter
+	basemessagingv1beta1.ChannelDefaulterSingleton = chDefaulter
```

Comment:
why:

```
baseeventingv1alpha1 "knative.dev/eventing/pkg/apis/eventing/v1alpha1"
	eventingv1alpha1 "knative.dev/eventing/pkg/apis/eventing/v1alpha1"
```
Why not just `eventingv1alpha1`?

Reply:
I was being dump :) I'll clean this up in the next round :)